doc/articles/guides/open-id-connect.md

```diff
@@ -18,7 +18,7 @@ This code uses the _IdentityServer_ demonstration endpoint with the following pa
 
 | Field     | Value                                     |
 | --------- | ----------------------------------------- |
-| Authority | `https://demo.itentityserver.io`          |
+| Authority | `https://demo.duendesoftware.com/`        |
```

Comment:
I know this is a legit but it might be worth making a note explaining the use of this demonstration endpoint (and why https://demo.itentityserver.io is no longer correct)

Reply:
The explanation is already present in the doc (line 26) and I personally feel that explaining the server name change to people who didn't realize it happened might be extra confusing information.